BAZProject/DetailMovieViper/DetailViper/DetailTableViewCell.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  DetailTableViewCell.swift
+//  BAZProject
+//
+//  Created by 1050210 on 13/02/23.
+//
+
+import UIKit
+
+class DetailTableViewCell: UITableViewCell{
+    
+    var presenter: DetailMoviePresenterProtocol?
+
+    @IBOutlet weak var detailNameLabel: UILabel!
+    @IBOutlet weak var detailCollectionView: UICollectionView!
+    var indexPath: Int?
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        detailCollectionView.delegate = self
```

Comment:
As far as I can see, this delegate is not needed, the protocol conformance is empty on line 55

Reply:
i need this for the UICollectionViewDelegateFlowLayout but i remove the extension of the collectionView delegate that extension is true that is unused